sample/.gitignore

```diff
@@ -0,0 +1,33 @@
+HELP.md
```

Comment:
Project name need to be more professional (`hello-world` or `spring-boot-start` instead of `sample`). This need not be changed now but keep in mind. Same goes for branch name. You can name it just `hello-world` (need not suffix with `branch`)